lib/sneakers/tasks.rb

```diff
@@ -6,8 +6,11 @@
 namespace :sneakers do
   desc "Start work (set $WORKERS=Klass1,Klass2)"
   task :run  => :environment do
+    puts "Forcing eager load"
+    ::Rails.application.eager_load!
 
     workers, missing_workers = Sneakers::Utils.parse_workers(ENV['WORKERS'])
+    puts "Forced eager load for #{workers}"
```

Comment:
do we need this puts?

Reply:
for now, we can remove later.  i just want to be able to see that my change is happening in logs.  otherwise there is no visible impact of eager loading